src/cclib/parser/utils.py

```diff
@@ -143,6 +165,29 @@ def __init__(self):
             self.number[self.element[i]] = i
 
 
+class Splitter:
+    """Split a line based not on a character, but a given number of field
+    widths.
+    """
+
+    def __init__(self, widths):
+        self.start_indices = [0] + list(accumulate(widths))[:-1]
+        self.end_indices = list(accumulate(widths))
+
+    def split(self, line):
+        """Split the given line using the field widths passed in on class
+        initialization.
+        """
+        elements = [line[start:end].strip()
+                    for (start, end) in zip(self.start_indices, self.end_indices)]
+        for i in range(1, len(elements)):
+            if elements[-1] == '':
```

Comment:
I don't quite understand why there's a elements[-1] here. Perhaps add a test case to illustrate this?